rules/0250-apache_rules.xml

```diff
@@ -61,9 +61,6 @@
     <description>Apache: Code Red attack.</description>
     <info type="link">http://www.cert.org/advisories/CA-2001-19.html</info>
     <info type="text">CERT: Advisory CA-2001-19 "Code Red" Worm Exploiting Buffer Overflow In IIS Indexing Service DLL</info>
-    <mitre>
-      <id>T1102</id>
-    </mitre>
     <group>automatic_attack,pci_dss_6.2,pci_dss_6.5.2,pci_dss_11.4,gdpr_IV_35.7.d,nist_800_53_SI.2,nist_800_53_SA.11,nist_800_53_SI.4,tsc_CC6.8,tsc_CC6.6,tsc_CC7.1,tsc_CC6.1,tsc_CC7.2,tsc_CC7.3,</group>
```

Comment:
For this one the tags T1190 "Exploit Public-Facing Application" and T107.001 "Application Layer Protocol: Web Protocols" may be correct